src/sql/display.py

```diff
@@ -92,6 +92,16 @@ def message_success(message):
     display(Message(message, style="color: green"))
 
 
+def message_warning(message):
+    """Display a success message"""
+    display(
+        Message(
+            message,
+            style="background-color:#fff3cd;color:#d39e00",
```

Comment:
is there any convention that we are following for these message styles, e.g., when should we use which style and how do we know which one to use for any future usecase?

Reply:
there's not explicit convention because this is subjective but what I do is show a message when a user operation succeeded and a warning when something went wrong but we can still proceed